src/components/views/messages/MLocationBody.tsx

```diff
@@ -206,14 +207,27 @@ function ZoomButtons(props: IZoomButtonsProps): React.ReactElement<HTMLDivElemen
     </div>;
 }
 
+/**
+ * Look up what map tile server style URL was provided in the homeserver's
+ * .well-known location, or, failing that, in our local config, or, failing
+ * that, defaults to the same tile server listed by matrix.org.
+ */
+export function findMapStyleUrl(): string {
+    return (
+        getTileServerWellKnown().map_style_url ??
+        SdkConfig.get().map_style_url ??
+        "https://api.maptiler.com/maps/streets/style.json?key=fU3vlMsMn4Jb6dnEIFsx"
```

Comment:
hardcoding an API key as fallback is unlikely to be great

Reply:
Nope, ignore me.  Re-reading our design, we should explicitly go to matrix.org and find the .well-known there if it's not defined on our homeserver.

Reply:
That's far from ideal from a privacy perspective. It should probably work the same as the default identity server, which you have to consent to use because you're connecting to a third party service. With maptiler surely by using the api you're going to need to consent to T&Cs as well, especially in Europe where gdpr applies.

Reply:
Even for connecting to matrix.org when you're a user of a different hs that is a privacy issue and one that we get complaints over in vector-im/element-web#11655

Reply:
Also if anything it should be defined in SdkConfig along with the other defaults.

Reply:
Thank you for raising this.  I have started discussing it with our product team.

Reply:
OK.  Initial discussion has concluded that we should remove the fallback to matrix.org and provide good documentation for server and client admins on how to set up the required config.

Reply:
So this comment is resolved in c51d7de3ad86ed300fdf78af8df815d5f6253725